src/translations/en.json

```diff
@@ -4180,7 +4180,7 @@
       "update_name": "Update {name}",
       "open_release_notes": "Open release notes",
       "create_backup": "Create backup before updating",
-      "description": "There is an update available for the {name}. You have {version} installed. Click update to update to version {newest_version}",
+      "description": "There is an update available for {name}. You have {version} installed. Click update to update to version {newest_version}",
```

Comment:
I think that we should split this in 2 separately translated pieces. 

- One per type that has an update available: "There is an update available for Home Assistant core." etc
- You have {version} installed. Click update to update to version {newest_version}."

Reply:
In that case, I think we should drop the first part ("There is an update available for the {name}. ") entirely.
We are already in an update context.

Reply:
People need to see _somewhere_ on the page what they are updating though.

Reply:
![image](https://user-images.githubusercontent.com/1444314/142804113-e88dfa4a-cb68-4291-a3ec-8895701707c7.png)

You're right. Let's drop it.